recipes/mapbox-wagyu/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.52.0"
+
+class MapboxWagyuConan(ConanFile):
+    name = "mapbox-wagyu"
+    description = "A general library for geometry operations of union, intersections, difference, and xor"
+    license = "BSD-2-Clause"
```

Comment:
```suggestion
    license = "LicenseRef-mapbox-wagyu"
```

The license is not a standard open source listed in SPDX. It contains parts of MIT, but the second paragraph is customized. 

Plus, in https://github.com/mapbox/wagyu/blob/master/include/mapbox/geometry/wagyu/almost_equal.hpp it has a BSD-3 customized too.